Makefile

```diff
@@ -1,5 +1,5 @@
 SRC_JS_FILES := $(shell find src/components src/js -type f -name '*.js')
-SRC_JS_FILES_FOR_COMPILER = $(shell sed -e ':a' -e 'N' -e '$$!ba' -e 's/\n/ --js /g' .build-artefacts/js-files | sed 's/^.*base\.js //')
+SRC_JS_FILES_FOR_COMPILER = $(shell sed -e ':a' -e 'N' -e '$$!ba' -e 's/\n/ --js /g' .build-artefacts/js-files | sed -r 's/(.*)/ --js \1/g')
```

Comment:
I don't think we don't  need this modification because that add  the base.js file to the single build file. And we don't need the base.js file.